pkg/infra/config.go

```diff
@@ -36,9 +36,7 @@ type LogConfig struct {
 func Defaults() Config {
 	return Config{
 		BSVNetwork: defs.NetworkMainnet,
-		DBConfig: DBConfig{
-			Engine: defs.DBTypeSQLite,
-		},
+		DBConfig:   *defs.DefaultDBConfig(),
```

Comment:
```suggestion
		DBConfig:   defs.DefaultDBConfig(),
```
Regarding above comment ☝️